chisel/src/dispatcher.rs

```diff
@@ -957,3 +960,11 @@ impl ChiselDispatcher {
         format!("{} {}", Paint::red(format!("{CHISEL_CHAR} Chisel Error:")), Paint::red(msg))
     }
 }
+
+#[inline]
+fn is_comment(input: &str) -> bool {
+    let trimmed_input = input.trim();
+    (trimmed_input.starts_with("//") &&
+        input.split_once('\n').map(|(_, nl)| nl.trim().is_empty()).unwrap_or(true)) ||
+        (trimmed_input.starts_with("/*") && trimmed_input.ends_with("*/"))
+}
```

Comment:
since this is easily testable, please add some sanity tests for this.
perhaps this should be solved with a regex instead?